xknx/dpt/dpt_1byte_control.py

```diff
@@ -0,0 +1,259 @@
+
+    @staticmethod
+    def _to_increment(value):
+        """Calculate the increment value from the stepcode and control bit as defined in the KNX standard section 3.3.1."""
+        # calculated using floor(100/2^((value&0x07)-1))
+        inc = [0, 100, 50, 25, 12, 6, 3, 1][value["step_code"] & 0x07]
+        return inc if value["control"] == 1 else -inc
+
+    @classmethod
+    def to_knx(cls, value, invert=False):
+        """Serialize to KNX/IP raw data."""
+        if not isinstance(value, int):
+            raise ConversionError("Cant serialize %s" % cls.__name__, value=value)
+
+        return super().to_knx(cls._from_increment(value), invert)
+
+    @classmethod
+    def from_knx(cls, raw, invert=False):
+        """Parse/deserialize from KNX/IP raw data."""
+        return cls._to_increment(super().from_knx(raw, invert))
+
+
+class DPTControlStepwiseDimming(DPTControlStepwise):
+    """Abstraction for KNX DPT 3.007 / DPT_Control_Dimming in stepwise mode."""
+
+    dpt_main_number = 3
+    dpt_sub_number = 7
+    value_type = "stepwise_dimming"
+
+
+class DPTControlStepwiseBlinds(DPTControlStepwise):
+    """Abstraction for KNX DPT 3.008 / DPT_Control_Blinds in stepwise mode."""
+
+    dpt_main_number = 3
+    dpt_sub_number = 8
+    value_type = "stepwise_blinds"
+
+
+class TitleEnum(Enum):
+    """Enum with a descriptive string representation.
+
+    Ensures values are rendered nicely, e.g. in home assistant.
+    """
+
+    def __str__(self):
+        """Return string representation."""
+        # pylint: disable=no-member
+        return self.name.title()
+
+
+class DPTControlStartStop(DPTControl):
+    """Abstraction for KNX DPT 3.xxx in start/stop mode."""
+
+    value_type = "startstop"
+    unit = ""
+
+    class Direction(TitleEnum):
+        """Enum for indicating the direction."""
+
+        DECREASE = 0
+        INCREASE = 1
+        STOP = 2
+
+    @classmethod
+    def to_knx(cls, value, invert=False):
+        """Convert value to payload."""
+        control = 0
+        step_code = 0
+        if value == cls.Direction(1):  # INCREASE/DOWN
+            control = 1
+            step_code = 1
+        elif value == cls.Direction(0):  # DECREASE/UP
+            control = 0
+            step_code = 1
+        elif value == cls.Direction(2):  # STOP
+            control = 0
+            step_code = 0
+        else:
+            raise ConversionError("Cant serialize %s" % cls.__name__, value=value)
+
+        values = {"control": control, "step_code": step_code}
+        return super().to_knx(values, invert)
+
+    @classmethod
+    def from_knx(cls, raw, invert=False):
+        """Convert current payload to value."""
+        values = super().from_knx(raw, invert)
+        if values["step_code"] == 0:
+            return cls.Direction(2)  # STOP
+        if values["control"] == 0:
+            return cls.Direction(0)  # DECREASE/UP
+        return cls.Direction(1)  # INCREASE/DOWN
```

Comment:
```suggestion
        if values["step_code"] == 0:
            return cls.Direction.STOP
        if values["control"] == 0:
            return cls.Direction.DECREASE
        return cls.Direction.INCREASE
```

Reply:
Not possible, for the same reason as above, `cls.Direction.DECREASE` is not defined when called from the child class `DPTControlStartStopBlinds`.